WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -258,8 +253,10 @@ private extension ProductFormViewController {
         let title = NSLocalizedString("Add more details",
                                       comment: "Title of the bottom sheet from the product form to add more product details.")
         let viewProperties = BottomSheetListSelectorViewProperties(title: title)
-        let dataSource = ProductFormBottomSheetListSelectorCommand(product: product,
-                                                                   isEditProductsRelease3Enabled: isEditProductsRelease3Enabled) { [weak self] action in
+        let actions = ProductFormBottomSheetActionsFactory.actions(product: product,
+                                                                   isEditProductsRelease2Enabled: isEditProductsRelease2Enabled,
+                                                                   isEditProductsRelease3Enabled: isEditProductsRelease3Enabled)
```

Comment:
I'd suggest making a private helper:

```swift
private func makeBottomSheetActions() -> [ProductFormBottomSheetAction] {
    ProductFormBottomSheetActionsFactory.actions(product: product,
                                                                   isEditProductsRelease3Enabled: isEditProductsRelease3Enabled) { [weak self] action in	                                                                   isEditProductsRelease2Enabled: isEditProductsRelease2Enabled,
                                                                   isEditProductsRelease3Enabled: isEditProductsRelease3Enabled).actions
}
```

And then we could use that in `updateMoreDetailsButtonVisibility` too.

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/2303/commits/768fd0feda273e33bc228e61eb61929a8b39d10d (I named it `createBottomSheetActions()` to be consistent with other `create*` functions)